src/classes/GE_GiftEntryController.cls

```diff
@@ -121,7 +121,7 @@ public with sharing class GE_GiftEntryController {
     /*******************************************************************************************************
     * @description Saves the data import record, dry runs the record, and returns updated
     * totals with the new row.
-    * @param dataImport: the NPSP_Data_Import__c record to update
```

Comment:
Since these functions are @AuraEnabled we need to know the names of the parameters in order to call them from LWC. Can we standardize on either `diRecord` or `dataImport` as the parameter name we use when passing in a data import record from LWC? I lean towards dataImport.

Reply:
@bdvorachek good point. My vote would be `dataImport` since that's more clear.

Reply:
I like "dataImport"!